.vscode/settings.json

```diff
@@ -1,42 +0,0 @@
-{
```

Comment:
This file is not meant to be removed. It's excluded from the template because it should not get copied over to projects, but it shares valid configurations for template development.

Reply:
But isn't this file doomed to deprecation as it won't be actively mantained as nobody really uses it?

Reply:
But you use it when you clone this repo...

Reply:
Ok, I understand :sweat_smile:

Reply:
But why not to rely on standard parameters for this project. Is there any customization required for it taking into account that this is only the template?

Reply:
Fixed :)

Reply:
> But why not to rely on standard parameters for this project. Is there any customization required for it taking into account that this is only the template?

Is just for working on the `doodba-copier-template` project itself. So it makes sense that is there.

Reply:
Yes, but that's why I ask the previous questions

Reply:
Well, they can be useful as a commonground for the doodba copier contributor, although a bunch of them are indeed outdated.